web/docs/data-model/prisma-file.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Prisma Schema File
+---
+
+import ImgWithCaption from '@site/blog/components/ImgWithCaption'
+
+Wasp uses [Prisma](https://www.prisma.io/) to interact with the database. Prisma is a "Next-generation Node.js and TypeScript ORM" that provides a type-safe API for working with your database.
+
+With Prisma, you define your application's data model in a `schema.prisma` file. 
+
+In Wasp, the `schema.prisma` file is located in your project's root directory:
+
+```c
+.
+├── main.wasp
+...
+// highlight-next-line
+├── schema.prisma
+├── src
+├── tsconfig.json
+└── vite.config.ts
+```
+
+ Wasp uses the `schema.prisma` file to understand your app's data model and generate the necessary code to interact with the database.
+
+## Wasp file and Prisma schema file
+
+Let's see how Wasp and Prisma files work together to define your application.
+
+Here's an example `schema.prisma` file where we defined some database options and two models (User and Task) with a one-to-many relationship:
+
+```prisma title="schema.prisma"
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id      Int        @id @default(autoincrement())
+  tasks   Task[]
+}
+
+model Task {
+  id          Int        @id @default(autoincrement())
+  description String
+  isDone      Boolean    @default(false)
+  user        User       @relation(fields: [userId], references: [id])
+  userId      Int
+}
+```
+
+Wasp reads this `schema.prisma` file and extracts the info about your database models and database config:
+
+<ImgWithCaption alt="Relationship between Wasp file and Prisma file" source="img/data-model/prisma_in_wasp.png" caption="Relationship between Wasp file and Prisma file"/>
+
+Wasp uses the `datasource` block to know which database you want to use (PostgreSQL in this case) and some other options.
+
+Wasp uses the `generator` block to generate the Prisma Client code that you can use in your application to interact with the database.
+
+Finally, Prisma models represent Wasp Entities which can be then used in the `main.wasp` file:
```

Comment:
```suggestion
Finally, Prisma models become Wasp Entities which can be then used in the `main.wasp` file:
```
`become` is maybe better? Not sure.

Reply:
This is the first moment where you introduce entities, in this doc, but it doesn't feel like detailed enough introduction. I know thi sis not super simple, but I would enrich it a bit. Explain that Wasp has the context of Entity, and these are data models in the context of Wasp, and that the way to define Entities is currently by defining Prisma models. And then also link to the part of docs where we explain it better. You could even maybe start the whole doc with this hm -> but ok maybe not, it can be more about how we use Prisma and then this we touch when we talk about models, ok.

Reply:
I'd rather link to the Entities page where we explained that bit 👍

Reply:
Ok let's link at least then -> I thought a bit of redunandcy might be a good fit here, but you do as you think is best!